client/src/views/Environment.tsx

```diff
@@ -34,36 +40,55 @@ function Environment() {
     setEndpoints(list);
   }, ENDPOINTS_POLLING_INTERVAL);
 
+
+    useInterval( async () => {
+      if (!environment || !currentContext) return;
+      const status = await okteto.status(environment.file, currentContext.name);
+       if (previousStatus && previousStatus === 'activating' && status === 'synchronizing' )
```

Comment:
We should not couple visualization with the business logic here. This call to the toast should be done in the `Environment.tsx` view, and leave to a single goal: refresh the status value.

Reply:
so should remove the business logic?

this will look like this: 
```
  const refreshStatus =  async () => {
    if (!environment || !currentContext) return;
    const status = await okteto.status(environment.file, currentContext.name);
      if (status === 'synchronizing' )
        window.ddClient.desktopUI.toast.success('env ready to use');
      setStatus(status);
  }
```

Reply:
can please clear this part:
@rlamana

Reply:
Refreshing the status value should be done in the context file, where the data is stored for the views to consume (just like we do with `refreshContext`. But the `window.ddClient.desktopUI.toast.success` and checking if the status has changed should be done in the Environment view.